knowledge-base/daterangepicker-css-customization.md

```diff
@@ -0,0 +1,97 @@
+---
+title: How to Customize the DateRangePicker
+description: Learn how to enhance the Blazor DateRangePicker by customizing its styling, disabling weekends, reducing the width of DateInputs, preventing horizontal expansion, hiding Start and End labels, and making the component more compact for better user experience and design consistency in your application.
+type: how-to
+page_title: How to Customize the DateRangePicker
+slug: daterangepicker-kb-css-customization
+tags: daterangepicker, styling
+ticketid: 1633582
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>DateRangePicker for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article answers the following questions:
+
+* How to customize the Blazor DateRangePicker styling?
```

Comment:
Use [shorter noun phrases](https://docs.telerik.com/style-guide/basic-rules-and-guidelines#noun-phrases) when possible.
```suggestion
* How to customize the styling of the Blazor DateRangePicker?
```